SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/DataBrokerProfileQueryOperationManager.swift

```diff
@@ -369,7 +369,7 @@ struct DataBrokerProfileQueryOperationManager: OperationsManager {
     // MARK: - Opt-Out Jobs
 
     internal func runOptOutOperation(for extractedProfile: ExtractedProfile,
-                                     on runner: WebJobRunner,
+                                     dependencies: DataBrokerOperationDependencies,
```

Comment:
The dependencies object is now passed to the manager – the other dependencies will start using the dependency manager in the next PR, which will refactor the operation manager.